melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/JournalpostSedKobling.java

```diff
@@ -31,4 +32,8 @@ public class JournalpostSedKobling {
 
     @Column(name = "sed_type")
     private String sedType;
+
+    public boolean erASed() {
+        return sedType.toUpperCase().startsWith("A");
```

Comment:
Samme spørsmål ang. nullsjekk, trengs det her?

Reply:
not null i db